modules/accessing-the-odh-dashboard.adoc

```diff
@@ -0,0 +1,24 @@
+:_module-type: PROCEDURE
+//pv2hash: 
```

Comment:
Similar comment to https://github.com/opendatahub-io/opendatahub-documentation/pull/55/files#r1386905171.